frontends/infinite-corridor/package.json

```diff
@@ -7,22 +7,29 @@
     "url": "https://github.com/mitodl/mit-open.git"
   },
   "dependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/material": "^5.9.0",
```

Comment:
These are the packages required for [`@mui`](https://github.com/mui/material-ui), see https://github.com/mitodl/hq/discussions/170

I originally thought we were already using an old version of this library, else I would have written the above discussions post sooner. My confusion stems from the fact that
- we do use `@material`
- `@mui` used to be called `@material-ui`

So I thought we were using the old version of `@mui`.